project/eScavenge/CLI.py

```diff
@@ -533,12 +532,27 @@ def setUp(self):
 
     def test_create_is_gm(self):
         self.assertEqual(None, self.cli.game)
-        self.assertEqual(game_create, self.cli.command("create"), "Failed to Create Game")
+        self.assertEqual(self.cli.command("create"), "Game Created")
```

Comment:
What exactly was changed here? It looks like the new code is a little backwards from the old code. The old code used a constant instead of hardcoded text, the order in the old code is better (const value, function call, return string on failure).